static/js/components/ShiftPage.jsx

```diff
@@ -0,0 +1,184 @@
+  shiftDelete: {
+    cursor: "pointer",
+    fontSize: "2em",
+    position: "absolute",
+    padding: 0,
+    right: 0,
+    top: 0,
+  },
+  shiftDeleteDisabled: {
+    cursor: "pointer",
+    fontSize: "2em",
+    position: "absolute",
+    padding: 0,
+    right: 0,
+    top: 0,
+    opacity: 0,
+  },
+}));
+
+const textStyles = makeStyles(() => ({
+  primary: {
+    fontWeight: "bold",
+    fontSize: "130%",
+  },
+  secondary: {
+    fontSize: "120%",
+    whiteSpace: "pre-wrap",
+  },
+}));
+
+export function shiftTitle(shift) {
+  if (!shift?.group.name) {
+    return (
+      <div>
+        <CircularProgress color="secondary" />
+      </div>
+    );
+  }
+
+  let result = `${shift?.group.name}`;
+  if (shift?.name) {
+    result += `: ${shift?.name}`;
+  }
+
+  return result;
+}
+
+export function shiftInfo(shift) {
+  if (!shift?.group.name) {
+    return (
+      <div>
+        <CircularProgress color="secondary" />
+      </div>
+    );
+  }
+  const startDate = new Date(`${shift.start_date}Z`).toLocaleString("en-US", {
+    hour12: false,
+  });
+  const endDate = new Date(`${shift.end_date}Z`).toLocaleString("en-US", {
+    hour12: false,
+  });
+
+  const array = [
+    ...(shift?.start_date ? [`Start Date: ${startDate}`] : []),
+    ...(shift?.end_date ? [`End Date: ${endDate}`] : []),
+  ];
+
+  // eslint-disable-next-line prefer-template
+  const result = array.join("\n");
+
+  return result;
+}
+
+const ShiftList = ({ shifts, deletePermission }) => {
+  const dispatch = useDispatch();
+  const deleteShift = (shift) => {
+    dispatch(shiftActions.deleteShift(shift.id)).then((result) => {
+      if (result.status === "success") {
+        dispatch(showNotification("Shift deleted"));
+      }
+    });
+  };
+
+  const classes = useStyles();
+  const textClasses = textStyles();
+  return (
+    <div className={classes.root}>
+      <List component="nav">
+        {shifts?.map((shift) => (
+          <ListItem button key={shift.id}>
+            <ListItemText
+              primary={shiftTitle(shift)}
+              secondary={shiftInfo(shift)}
+              classes={textClasses}
+            />
+            <Button
+              key={shift.id}
+              id="delete_button"
+              classes={{
+                root: classes.shiftDelete,
+                disabled: classes.shiftDeleteDisabled,
+              }}
+              onClick={() => deleteShift(shift)}
+              disabled={!deletePermission}
+            >
+              &times;
+            </Button>
+          </ListItem>
+        ))}
+      </List>
+    </div>
+  );
+};
+
+const ShiftPage = () => {
+  const classes = useStyles();
+  const { shiftList } = useSelector((state) => state.shifts);
+  const currentUser = useSelector((state) => state.profile);
+
+  if (!shiftList) {
+    return <CircularProgress />;
+  }
+
+  const permission =
+    currentUser.permissions?.includes("System admin") ||
+    currentUser.permissions?.includes("Manage groups") ||
```

Comment:
It's not clear that we want to grant this access to someone with `manage groups` -- since we have a more fine grained `manage shifts` if someone needs to be able to manage shifts then they should get that permissions added to their user